ComputerVision/Rest/AnalyzeImage/Program.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace analyzeImage
+{
+    static class AnalyzeImage
+    {
+        // Replace <Subscription Key> with your valid subscription key.
+        const string subscriptionKey = "0123456789abcdef0123456789ABCDEF";
+
+        // You must use the same Azure region in your REST API method as you used to get your subscription keys. 
+        const string uriBase = "https://westus.api.cognitive.microsoft.com/vision/v2.0/analyze";
+
+        static void Main()
+        {
+            Console.WriteLine("Detect objects in images:");
+
+            string imageFilePath = @"Images\sample6.png";
+            string remoteImageUrl = "https://github.com/harishkrishnav/cognitive-services-dotnet-sdk-samples/raw/master/ComputerVision/Images/sample4.png";
+
+            Console.WriteLine("Images being analyzed ...");
+            var t1 = AnalyzeFromStreamAsync(imageFilePath);
+            var t2 = AnalyzeFromUrlAsync(remoteImageUrl);
+
+            Task.WhenAll(t1, t2).Wait(5000);
+            Console.WriteLine("Press ENTER to exit");
+            Console.ReadLine();
+        }
+        
+        static async Task AnalyzeFromUrlAsync(string imageUrl)
+        {
+            if (!Uri.IsWellFormedUriString(imageUrl, UriKind.Absolute))
+            {
+                Console.WriteLine("\nInvalid remote image url:\n{0} \n", imageUrl);
+                return;
+            }
+
+            try
+            {
+                HttpClient client = new HttpClient();
+
+                // Request headers.
+                client.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
+
+                // Request parameters. A third optional parameter is "details".
+                // Comment parameters that aren't required
+                string requestParameters = "visualFeatures=" +
+                    "Categories," +
+                    "Description," +
+                    "Color, " +
+                    "Tags, " +
+                    "Faces, " +
+                    "ImageType, " +
+                    "Adult , " +
+                    "Brands , " +
+                    "Objects"
+                    ;
+
+                // Assemble the URI for the REST API method.
+                string uri = uriBase + "?" + requestParameters;
+
+                HttpResponseMessage response;
+                string requestBody = " {\"url\":\"" + imageUrl + "\"}";
+                var content = new StringContent(requestBody);
+                content.Headers.ContentType = new MediaTypeHeaderValue("application/json");
+                response = await client.PostAsync(uri, content);
```

Comment:
Let's add a blank line here, and also prefix it with a comment like // Post the request and display the result.

Please do this for all the Rest samples, so we make the Post step stand out a bit more.